<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1311</width>
      <height>851</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>工作界面</string>
     </attribute>
     <widget class="QPushButton" name="btn_motion">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>280</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: qradialgradient(spread:repeat, cx:0.5, cy:0.5, radius:0.077, fx:0.5, fy:0.5, stop:0 rgba(0, 169, 255, 147), stop:0.497326 rgba(0, 0, 0, 147), stop:1 rgba(0, 169, 255, 147));
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>71</width>
        <height>71</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="v_edit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>速度Y</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_vset">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dset">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>速度X</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="d_edit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_gmset">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>步态模式 </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="gm_edit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>220</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>02</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_tguset">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>280</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>周期</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tgu_edit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>280</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_pset">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>340</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>340</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>下压</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="p_edit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>340</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_connect">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>130</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>71</width>
        <height>71</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_refresh">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>170</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="PBT_Open">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>480</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>71</width>
        <height>71</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>0</y>
        <width>640</width>
        <height>400</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(52, 101, 164);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>410</y>
        <width>640</width>
        <height>400</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(115, 210, 22);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="PBT_Close">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>620</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>71</width>
        <height>71</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_status">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>500</y>
        <width>311</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>461</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>显示</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>速度Z</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="VZ_edit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_vsetZ">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 12pt &quot;Adobe 宋体 Std L&quot;;
border-color: rgb(0, 255, 0);
color:rgb(255,0,0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>串口</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>60</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>serialport choose</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>480</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Serial port operation</string>
      </property>
     </widget>
     <widget class="QLabel" name="accept_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>accept</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>386</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Parity check</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbx_setPortName">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>80</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cbx_setBaudRate">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>150</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>8</number>
      </property>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>76800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>128000</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>460800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>961200</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btn_clear_rx">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>540</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_clear_tx">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>490</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_HexShow">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>300</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>HEX_Show</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbx_setStopBits">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>210</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>300</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Data bits</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_send">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>420</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>send</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_rx">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>781</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_tx">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>420</y>
        <width>641</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>150</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Baud rate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_open_close">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>540</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>open serialport</string>
      </property>
     </widget>
     <widget class="QLabel" name="send_label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>send</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_timeStamp">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>612</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>time stamp</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>220</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop bit</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbx_setParity">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>390</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>none</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Odd Check</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Even Check</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cbx_setDataBits">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>290</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="checkBox_HexSend">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>580</y>
        <width>73</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>HEX_Send</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1311</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
